### PR TITLE
Add puppet-strings docs

### PR DESCRIPTION
**Pull request by baurmatt**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -1,35 +1,11 @@
...
-# === Examples
-#
-#  Installing .NET 4.5
-#
-#    dotnet { 'dotnet45':
-#      version => '4.5',
-#    }
-#
+# @summary Module to install the Microsoft .NET framework on Windows.
+# @param version The version of .NET to be managed.
```
Comment (alexjfisher) [inline manifests/init.pp:2]: "Separate comment sections (such as @summary, @example, or the @param list) with a blank comment line (that is, a # with no additional content), to improve readability."
https://puppet.com/docs/puppet/5.5/puppet_strings_style.html

File: manifests/params.pp [hunk 2]
```diff
@@ -1,12 +1,4 @@
-# Author::    Liam Bennett (mailto:liamjbennett@gmail.com)
-# Copyright:: Copyright (c) 2014 Liam Bennett
-# License::   MIT
-
-# == Class dotnet::params
-#
-# This class is meant to be called from `dotnet`
-# It sets variables according to platform
-#
+# @summary This class is meant to be called from `dotnet`. It sets variables according to platform.
```
Comment (alexjfisher) [inline manifests/params.pp:1]: This wants an `# @api private`

**Review by alexjfisher — CHANGES_REQUESTED** [review]

**Comment by baurmatt** [comment]
Solved merge conflict and added `# @api private` to params.pp

Sorry it took me that long 🙈

**Review by smortex — COMMENTED** [review]
LGTM, but REFERENCE.md needs to be updated to reflect the `@api private` change :wink:

**Comment by baurmatt** [comment]
Of course 🙈 Ran `bundle exec rake strings:generate:reference`.

**Review by smortex — APPROVED** [review]
LGTM

**Pull request merged**